documentation/tools/automation.md

```diff
@@ -0,0 +1,20 @@
+# How to automate simple checks
+
```

Comment:
Not sure if it is best to include here...but the team needs to be clear what "linting rules" are required and shouldn't be "overridden" with tooling or comments. Case in point the "markdown-link-check" that is currently blocking this PR...is that a "comment workaround" or fixing the link? The team should have consensus on when to override tooling rules.

Reply:
Thank you, I added your comments to the document, as I think they are relevant to the content